pyproject.toml

```diff
@@ -0,0 +1,47 @@
+[tool.poetry]
+name = "pycytominer"
+version = "1.0.0"
+description = "Processing perturbation profiling readouts."
```

Comment:
Consider aligning the value here with that of the Github repository description.

Reply:
Done